app/src/main/java/com/futurice/freesound/network/api/model/RequestHelper.java

```diff
@@ -20,7 +20,7 @@
 
 import android.support.annotation.NonNull;
 
-import java.util.Iterator;
+import kotlin.collections.CollectionsKt;
```

Comment:
No good reason, this PR just started as a demo of calling the Kotlin stdlib from Java.